src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageUiTest.java

```diff
@@ -465,6 +464,45 @@ public void testViewPhotoAndAjaxForLargeScaledSession() throws Exception {
         resultsPage.hoverClickAndViewStudentPhotoOnHeading("1-2", Const.SystemParams.DEFAULT_PROFILE_PICTURE_PATH);
     }
 
+    @Test
+    public void testViewNoSpecificSection() {
+        ______TS("No Specific Section shown for recipient > question > giver with General Feedback");
+        resultsPage = loginToInstructorFeedbackResultsPageWithViewType("CFResultsUiT.instr", "Open Session", true,
+                "recipient-question-giver");
+        assertTrue(resultsPage.isSectionPanelExist(Const.NO_SPECIFIC_SECTION));
+
+        ______TS("No Specific Section shown for recipient > giver > recipient with General Feedback");
```

Comment:
Can you group your test cases in w.r.t viewType? It would be more readable :)

Reply:
I can't. Because it depends on which session I signed into. Unless I keep signing in to different sessions for each assertions, but that will take longer for performance?